docs/en/operations/settings/settings.md

```diff
@@ -254,6 +273,27 @@ Possible values:
 
 Default value: 1.
 
+## date_time_input_format {#settings-date_time_input_format}
+
+Enables or disables extended parsing of date and time formatted strings.
+
+The setting doesn't apply to date and time functions.
+
+Possible values:
+
+- `'best effort'` — Enables extended parsing.
+
+    ClickHouse can parse all the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) data and time formats. For example, `'2018-06-08T01:02:03.000Z'`.
+
+- `'basic'` — Use basic parser.
+
+    ClickHouse can parse only the basic format `YYYY-MM-DD HH:MM:SS`.
```

Comment:
it's not clear if "best effort" supports basic format as well or not (space instead of T makes it non-ISO8601)